kubejs/server_scripts/mods/modern_industrialization/packer.js

```diff
@@ -42,6 +43,90 @@ ServerEvents.recipes(e => {
         e.custom(newRecipe).id(id);
     }
 
+    // -- FLUID PIPE -- //
+    packer(
+        st('fluid_pipe'),
+        4,
+        200,
+        [
+            { amount: 1, tag: 'modern_industrialization:fluid_pipes' },
```

Comment:
would prefer the use of the utility function here even though its a tag.
`mi('fluid_pipes')` will still return the same thing as you have here.